LayoutTests/platform/glib/http/tests/resourceLoadStatistics/website-data-removal-for-site-without-user-interaction-expected.txt

```diff
@@ -0,0 +1,15 @@
+Check that website data gets removed after a period of no user interaction.
+
+Before deletion: Client-side cookie exists.
+Before deletion: HttpOnly cookie exists.
+Before deletion: Regular server-side cookie exists.
+Before deletion: LocalStorage entry does exist.
+Before deletion: IDB entry does exist.
+
+After deletion: HttpOnly cookie exists.
+After deletion: Client-side cookie exists.
```

Comment:
Is this expected?

Reply:
Same thing here?

Reply:
Yes, sorry that wasn't clear, this is a glib expectation and the old behavior is expected.